aries_cloudagent/core/conductor.py

```diff
@@ -467,10 +472,24 @@ async def outbound_message_router(
                 outbound.reply_from_verkey = inbound.receipt.recipient_verkey
             # return message to an inbound session
             if self.inbound_transport_manager.return_to_session(outbound):
-                return OutboundSendStatus.SENT_TO_SESSION
+                await profile.notify(
```

Comment:
Not returning here will cause unexpected behavior

Reply:
Not the not returning a value but the short circuiting of the method.

Reply:
got it.